extensions/amp-story/1.0/amp-story.css

```diff
@@ -178,8 +178,8 @@ amp-story-page[active] {
   z-index: 1 !important;
 }
 
-amp-story amp-sidebar[side][open] {
-  background-color: #ffffff !important;
+amp-sidebar[side][open] {
+  background-color: #ffffff;
```

Comment:
+1, but maybe the sidebar should also be white all the time, not just when the `[open]` attribute is set. The sidebar can be visible on the screen without the `[open]` attribute when the sidebar is animating towards its closed state.

Reply:
Actually, I'm also realizing that we might need to lower the specificity for the selector which sets the `background-color`, to let users override this without having to outdo our specificity.

Reply:
Since it's overridable, I moved it to `amp-story-user-overridable.css` and also changed the selector to just `amp-sidebar[side]`. This is to override the original background color of[ light grey set by the sidebar component](https://github.com/ampproject/amphtml/blob/master/extensions/amp-sidebar/0.1/amp-sidebar.css#L23).